cms/static/sass/programs/_app-typography.scss

```diff
@@ -0,0 +1,80 @@
+
+// ------------------------------
+// Programs: App Fonts
+
+// About: resolving issue with rem units when html font-size is not 100%.
+// the issue is that the unit being used for fonts, padding and margins in the
+// UX Pattern Library (UXPL) is relative to the font-size set on the html element.
+// In studio this is set to 62.5%, however the UXPL assumes it is set to 100%. Because
+// the page in studio has other elements that are native to studio that need to be styled
+// we cannot adjust the html element's font-size on this page so I have added a Sass partial
+// that sets px values for the elements to resolve the issue. In the long-term the UXPL will
+// be added to studio and we will be able to remove this partial.
+.program-app {
+  padding-top: 20px;
```

Comment:
Nitpick: this Sass file uses indentation of two spaces, others in this PR use four. I think our standard in edx-platform is two spaces, so it would be nice to stick to that.